common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -200,7 +200,8 @@ object ContentWidths {
 
   object MinuteMedia extends ContentRelation {
     override val inline = WidthsByBreakpoint(
-      mobile = Some(95.vw),
+      // In the minute inline images match the immersive header size
+      mobile = Some(1000.px),
```

Comment:
Nope, it's like the immersive header where the ratio stays the same, so wider than viewport but editors select images that crop into the middle.